### ISSUE TITLE
Dependencies install instruction

### ISSUE THREAD

**Issue by rht** [rauner/air-pollution#1]
I'm looking to run the code in this repo. I had to install the dependencies by hand because there are no explicit instruction.
What I have tried:
- `install.packages('dplyr')`
- `install.packages('tidyr')`
- `install.packages("remotes")`
- `apt install libgdal-dev`

But I'm stuck at installing the `moinput` dependency. It is not available in the default mirror. I assume that it is https://github.com/piklev/moinput, but trying `install.packages("moinput", repos="https://rse.pik-potsdam.de/r/packages")
` simply hanged.

I am not an R developer, and so don't know the proper way to set this up. Any help would be appreciated!

**Comment by rauner** [comment]
Hi there, you are def. on a good path to getting the model run. Unfortunately, the moinput dependency is not actively developed anymore. The new package is https://github.com/pik-piam/mrcommons. You can try to change the functions from moinput to mrcommons. However, a more promising approach is to ask for the current model implementation: https://www.pik-potsdam.de/members/davidba/homepage-de

**Comment by rht** [comment]
Thank you for the swift reply!

I tried `install.packages("mrcommons", repos="https://rse.pik-potsdam.de/r/packages")`, but this too didn't end well
```
install.packages("mrcommons", repos="https://rse.pik-potsdam.de/r/packages")
Installing package into ‘/usr/local/lib/R/site-library’
(as ‘lib’ is unspecified)
Warning: dependencies ‘hdf5r’, ‘openxlsx’, ‘bezier’ are not available
also installing the dependencies ‘mrdrivers’, ‘GDPuc’

trying URL 'https://rse.pik-potsdam.de/r/packages/src/contrib/mrdrivers_1.3.0.tar.gz'
Content type 'application/x-gzip' length 71683 bytes (70 KB)
==================================================
downloaded 70 KB

trying URL 'https://rse.pik-potsdam.de/r/packages/src/contrib/GDPuc_0.11.1.tar.gz'
Content type 'application/x-gzip' length 950040 bytes (927 KB)
==================================================
downloaded 927 KB

trying URL 'https://rse.pik-potsdam.de/r/packages/src/contrib/mrcommons_1.33.4.tar.gz'
Content type 'application/x-gzip' length 1143576 bytes (1.1 MB)
==================================================
downloaded 1.1 MB

Warning in untar2(tarfile, files, list, exdir, restore_times) :
  using pax extended headers
* installing *source* package ‘GDPuc’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
*** copying figures
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (GDPuc)
Warning in untar2(tarfile, files, list, exdir, restore_times) :
  using pax extended headers
ERROR: dependency ‘bezier’ is not available for package ‘mrdrivers’
* removing ‘/usr/local/lib/R/site-library/mrdrivers’
Warning in untar2(tarfile, files, list, exdir, restore_times) :
  using pax extended headers
ERROR: dependencies ‘mrdrivers’, ‘hdf5r’, ‘openxlsx’ are not available for package ‘mrcommons’
* removing ‘/usr/local/lib/R/site-library/mrcommons’
```

**Comment by rauner** [comment]
Can you try installing the dependencies manually that are not available, '‘bezier’', ‘mrdrivers’, ‘hdf5r’, ‘openxlsx’?

**Comment by rht** [comment]
It looks like those packages are not available in https://rse.pik-potsdam.de/r/packages, but I was able to install them (except for openxlsx) via the default mirror (i.e. if I don't specify the repo at all). With openxslx, I had to do `remotes::install_github("ycphs/openxlsx")`. With those, I was able to execute calcMonetization.R till completion. I have another question, but I will open a separate issue for that.

Another note, I saw this message:
```
The legacy packages maptools, rgdal, and rgeos, underpinning the sp package,
which was just loaded, will retire in October 2023.
Please refer to R-spatial evolution reports for details, especially
https://r-spatial.org/r/2023/05/15/evolution4.html.
```
It looks like the R environment is hard to future proof.
